src/go/k8s/tests/e2e/update-image-tls/01-assert.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: apps/v1
```

Comment:
In this test we should assert Issuer and Certificate

Reply:
we're already doing that in produce-tls, I think it's enough in one test. There's nothing special in the upgrade scenario, this aims to test that serama can talk to nodes with tls certs

Reply:
Ok